packages/server/src/main/utils.ts

```diff
@@ -1,7 +1,8 @@
+const crypto = require('crypto');
 const alphabet = '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ';
 
 export function randomString(length: number) {
   let result = '';
-  for (let i = length; i > 0; --i) result += alphabet[Math.floor(Math.random() * alphabet.length)];
+  for (let i = length; i > 0; --i) result += alphabet[crypto.randomInt(0, alphabet.length)];
```

Comment:
1. Is there a reason to count down from ``length`` instead of counting up? Seems uncommon.
2. ``0`` is the default ``min`` value and can be omitted
```suggestion
  for (let i = 0; i < length; i++) result += alphabet[crypto.randomInt(alphabet.length)];
```

Reply:
👍